dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -92,13 +92,13 @@ type ButtonProps = {
 export const FollowNotificationsButton = ({
 	isFollowing,
 	onClickHandler,
-	cssOverrides,
-}: ButtonProps & { cssOverrides?: SerializedStyles }) => {
+	withExtraBottomMargin = false,
+}: ButtonProps & { withExtraBottomMargin?: boolean }) => {
 	return (
 		<button
 			onClick={onClickHandler}
 			type="button"
-			css={[buttonStyles, cssOverrides]}
+			css={[buttonStyles(withExtraBottomMargin)]}
```

Comment:
Not a blocker, but I was thinking something more along the lines of a static `css` declaration:

```ts
			css={[buttonStyles, withExtraBottomMargin && bottomMarginStyles]}
```